games-util/pyfa/files/pyfa-2.9.3-import-pyfa.patch

```diff
@@ -0,0 +1,28 @@
+From c12e4c1358025940aa1fbdf0e17f85e3821e7230 Mon Sep 17 00:00:00 2001
+From: Andreas Zuber <zuber@puzzle.ch>
+Date: Mon, 30 Oct 2017 09:52:32 +0100
+Subject: [PATCH] import pyfa
+
+Append $(python_get_sitedir)/pyfa to import path
+
+Ensures that pyfa's import statements continue to work for systemwide
+installation.
+---
+ pyfa.py | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/pyfa.py b/pyfa.py
+index d73ef097..dbf73db2 100755
+--- a/pyfa.py
++++ b/pyfa.py
+@@ -22,6 +22,7 @@
+ import datetime
+ import os
+ import sys
++sys.path.append(os.path.join("%%SITEDIR%%", "pyfa"))
```

Comment:
this seems like a weird hack. Are there modules that do

    import foo

instead of

    import pyfa.foo

?

Reply:
Yes, the application isn't usually installed globally as a python packages so their own modules are loaded relative. If we don't add that path it will not find them.

Reply:
Is there any value to installing them into sitedir then? Maybe you should put them in `/usr/lib/pyfa` or sth like that.

Reply:
Sorry for the late reply, was on vacation and did not have time to look into it until now. I could do that, since I took the ebuild over from someone else I always assumed this is the standard way of installing python applications.
I can change that and simply install it into its own directory. What would you prefer and why?

Reply:
I'm going to merge it as-is today but yes, I think installing it into dedicated directory would be better, especially if it could save us from having that patch.